fin_control/msg/SetAngles.msg

```diff
@@ -1,6 +1,6 @@
 uint16  MAX_ANGLE = 40
-float32 f1_angle_in_radians
-float32 f2_angle_in_radians
-float32 f3_angle_in_radians
-float32 f4_angle_in_radians
+float64 f1_angle_in_radians
+float64 f2_angle_in_radians
+float64 f3_angle_in_radians
+float64 f4_angle_in_radians

```

Comment:
is it necessary to update [angle_plus_offset](https://github.com/ChrisScianna/ROS-Underwater-RnD/blob/26d181824aeff37f599778702821ca0922629867/fin_control/src/fin_control.cpp#L265) variable type?

Reply:
Good observation. Thing is, the dynamixel workbench API uses `float`. We have to bridge that in `fin_control`. I'd suggest we do that in a follow-up PR. WDYT?

Reply:
SGTM